src/bare-metal/aps/safemmio/using.md

```diff
@@ -0,0 +1,15 @@
+# Using It
```

Comment:
It looks like `better-uart/using.md` needs removed?

Reply:
You're right, thanks. Done.